examples/vhdl/sqrt/sqrt.toml

```diff
@@ -1,15 +1,24 @@
+[[design]]
```

Comment:
This project toml didn't run without these changes.

Reply:
I don't recall but let's chalk it up to operator error and then revisit after merge (if it comes up).